flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+    "only-arches": ["x86_64"],
```

Comment:
so aarch64 users are left out?

Reply:
The AppImage itself is for x86_64, is there an aarch64 AppImage available as far as you know?

Reply:
you tell me. I'm not the one packaging this. I'm just reviewing.

building from source is also an option, albeit because it's Electron, it increases the challenge level.